docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-portal-product-offer-management-feature-walkthrough.md

```diff
@@ -12,11 +12,16 @@ To learn more about the feature and to find out how end users use it, see [Marke
 
 {% endinfo_block %}
 
+## Module dependency graph
```

Comment:
Please provide a more detailed description of the feature.
E.g. which data of product offer we can manage, prices, validity dates etc.